app/serializers/institution_profile_serializer.rb

```diff
@@ -105,7 +105,7 @@ def yellow_ribbon_programs
   end
 
   def versioned_school_certifying_officials
-    object.school_certifying_officials.map do |sco|
+    object.school_certifying_officials.select { |sco| sco.institution_id == object.id }.map do |sco|
```

Comment:
🤔  I'd expect `school_certifying_officials` association with `institutions` to be based on `institution_id` already.
Why is it based on `facility_code`?

Reply:
That is a lot cleaner to update the association for institution and remove the mapping. I updated this in a commit I have pushed. Look it over when you get a chance and let me know what you think.